script.sh

```diff
@@ -0,0 +1,6 @@
+
+echo Compiling...
+g++ bulkingDiet.cpp cuttingDiet.cpp main.cpp -o run.o
+echo -----------Ready-------------
+./run.o
+rm -r run.o
```

Comment:
Temporary fix - will include an if statement to check for the build when you run the script, else will build and run.